### ISSUE TITLE
Quantumplots: Inability to load amsmath to provide commands needed for captions

### ISSUE THREAD

**Issue by alastair-abbott** [cgogolin/quantum-journal#110]
The quantumplots implementation doesn't load commonly used packages such as amsmath or amssymb that are often necessary in math mode for captions (e.g., \text{}). 

I tried adding the appropriate \usepackage{} commands to the mplstyle file but they had no effect.

**Comment by cgogolin** [comment]
@johannesjmeyer , could you have a look at this, please?

**Comment by johannesjmeyer** [comment]
Hi @alastair-abbott, would you be so kind to send a minimal example of the error? Normally adding the commands to the file should work. You could also try modifying `rcParams` directly:
```python
plt.rcParams["pgf.preamble"] = plt.rcParams["pgf.preamble"] + r" \usepackage{amsmath}"
```

In the meantime, I created a new package that holds the same functionality as quantumplots but is also compatible with RevTeX (and easier to use). You might want to have a look at it. It is found here: https://github.com/johannesjmeyer/rsmf

**Comment by alastair-abbott** [comment]
Hi @johannesjmeyer, I tried to replicate my previous issue but it seems it works now, sorry about that! (I wonder if something was up with my jupyter installation/configuration, as I've reinstalled conda and everything since then.)

Having said that, I think it would be good to either a) document how to load extra packages, etc., that one might need to use in plotting, or preferably b) make an easier way to do this than modifying the mplstyle file. Indeed, I don't think it's particularly uncommon that people may need to use packages such as amsmath, or others, in order to typset the text they need.

Thanks for the heads up about your rsmf package – it seems very useful and I found it much easier to use than quantumplots. I found the way quantumplots is distributed rather unintuitive: what is distributed is essentially just an example file. If one actually wishes to use it, then in practice one has to either delete all of the actual examples or copy the definitions and setup functions into the notebook one wishes to use to make the plots for a paper. A package that one can simply import and use, like rsmf, seems a much smoother workflow.

**Comment by johannesjmeyer** [comment]
Hi @alastair-abbott, thanks for coming back to this. 

> Having said that, I think it would be good to either a) document how to load extra packages, etc., that one might need to use in plotting, or preferably b) make an easier way to do this than modifying the mplstyle file. Indeed, I don't think it's particularly uncommon that people may need to use packages such as amsmath, or others, in order to typset the text they need.

You can always effect such a modification by appending to `rcParams["pgf.preamble"]` directly without having to modify the `.mplstyle` file. But I totally agree that such an option should be added (I would add it to `rsmf`though).

> Thanks for the heads up about your rsmf package – it seems very useful and I found it much easier to use than quantumplots. I found the way quantumplots is distributed rather unintuitive: what is distributed is essentially just an example file. If one actually wishes to use it, then in practice one has to either delete all of the actual examples or copy the definitions and setup functions into the notebook one wishes to use to make the plots for a paper. A package that one can simply import and use, like rsmf, seems a much smoother workflow.

I also agree with that. I also found the workflow quite clumsy, which is why I created the `rsmf` package in the first place.
@cgogolin, what do you think about removing `quantum-plots` and advertising `rsmf` instead?

**Comment by cgogolin** [comment]
This will be addressed in #126  via #124
